<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableColumnCreater</class>
 <widget class="QDialog" name="TableColumnCreater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание колонки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonAceept">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выбор типа колонки</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEditColumn"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxColumn"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
